File: pset1/cash/cash.c
#include<stdio.h>
#include<cs50.h>
#include<math.h>

int main(void)

{

    float n;
    do
    {

        n = get_float("Change owned:");

    }
    while (n <= 0);

    int cents = round(n * 100);

    int result25 = cents / 25;
    int reminder25 = cents % 25;

    int result10 = reminder25 / 10;
    int reminder10 = reminder25 % 10;

    int result5 = reminder10 / 5;
    int reminder5 = reminder10 % 5;

    int result1 = reminder5 / 1;


    printf("%i\n", result25 + result10 + result5 + result1);

}

File: pset6/cash/cash.py
from cs50 import get_float



n = get_float("Change owned: ")

if n > 0:

    cents = round(n * 100)

    result25 = int(cents / 25)
    reminder25 = cents % 25

    result10 = int(reminder25 / 10)
    reminder10 = reminder25 % 10

    result5 = int(reminder10 / 5)
    reminder5 = reminder10 % 5

    result1 = int(reminder5 / 1)

    s = result25 + result10 + result5 + result1

    print(s)
    


